hazelcast/src/main/resources/hazelcast-config-5.1.json

```diff
@@ -1463,6 +1463,53 @@
                 }
               }
             }
+          },
+          "tiered-store": {
```

Comment:
Please add some tests to the newly added config section. I suggest putting them into a new directory at `src/test/resources/com/hazelcast/config/yaml/testcases/member/tiered-store`. Please find the details [on the wiki](https://github.com/hazelcast/hazelcast/wiki/Checklist-for-Hazelcast-config-changes#yaml-configuration-schema-update).

Reply:
thank you for the comment. I added some tiered-store yaml validation tests.